pulsar-client-cpp/include/pulsar/Schema.h

```diff
@@ -27,6 +27,25 @@
 
 namespace pulsar {
 
+/**
+ *  Encoding types of supported KeyValueSchema for Pulsar messages.
+ */
+enum KeyValueEncodingType
+{
+    /**
+     * Key is stored as message key, while value is stored as message payload.
+     */
+    SEPARATED,
+
+    /**
+     * Key and value are stored as message payload.
+     */
+    INLINE
+};
+
+// Return string representation of result code
+PULSAR_PUBLIC const char *strEncodingType(KeyValueEncodingType schemaType);
```

Comment:
Override `operator<<` rather than providing another function. See how we deal with `Result`. The `strResult` function is a legacy function.